edx_name_affirmation/tasks.py

```diff
@@ -187,3 +260,81 @@ def proctoring_update_verified_name(
                     attempt_id=attempt_id,
                 )
             )
+
+
+@shared_task(
+    bind=True, autoretry_for=(Exception,), default_retry_delay=DEFAULT_RETRY_SECONDS, max_retries=MAX_RETRIES,
+)
+@set_code_owner_attribute
+def proctoring_update_verified_name_task(
```

Comment:
Same as above, added new proctoring task to allow us to change the signature. For the new proctoring task, we no longer need the proctored exam data to be passed in to the task.